packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -4,12 +4,14 @@
 
 import { AuthServerError } from 'fxa-auth-client/browser';
 
-export type AuthUiError = AuthServerError;
+export type AuthUiError = AuthServerError & {version?: number};
```

Comment:
I think in the React epic, we could create an issue to consolidate our `auth-errors` (if you have a doc going yet for planning you could just note it there 🤷‍♀️). Looks like we have the exact same error messaging in `auth-errors.js` in content-server using gettext, then we have a repeat here for Settings? We probably just made a new file for Settings with the intent to share later but we can't until content-server is using Fluent. Does that sound right to you?

Reply:
strong agree!! we have the same strings repeated in a few places and it would be great to get this shared. the mystery test that was breaking was the same issue also -- in the test setup, we were checking for a specific state by checking the string for the error message. if that had been pointing to a shared const, it would have been fine!